.gitignore

```diff
@@ -295,11 +295,16 @@ tmp-php.ini
 
 # ------------------------------------------------------------------------------
 # Special cases to invert previous ignore patterns
-# ------------------------------------------------------------------------------
+# ----------gitp--------------------------------------------------------------------
 !/ext/bcmath/libbcmath/src/config.h
 !/ext/fileinfo/libmagic/config.h
 !/ext/fileinfo/libmagic.patch
 !/ext/fileinfo/magicdata.patch
 !/ext/mbstring/oniguruma.patch
 !/ext/pcre/pcre2lib/config.h
 !/win32/build/Makefile
+
+# ------------------------------------------------------------------------------
+# Clion IDE files
+# ------------------------------------------------------------------------------
+.idea/**
```

Comment:
Hello, thanks for the patches! Very nice of you to get to this part. What was meant in the previous comment was to omit this change :) Because `/.idea/` is only one of the folders that we should then ignore. Visual studio Code for example uses `/.vscode/` and so on... It can quickly increase the .gitignore even more what currently already is. I'm not sure if there would be performance impact on these editors and command line with too long .gitignore file but if not anything for the readability. In any case, you're welcome to sen a separate pull request concerning this change only. Thanks.

Reply:
> What was meant in the previous comment was to omit this change :)

ACK. I suggest to configure the Git client to ignore IDE files – this can also be done globally (i.e. for all Git repos on the machine).